### DISCUSSION TITLE
how to config vscode markdown lsp in coc?

### DISCUSSION THREAD

**Discussion by WillDawnlll** [neoclide/coc.nvim#4084] [Q&A]
recently i noticed vscode markdown lsp can work with other editor , so i try to set it up with coc in vim .
but after some google i cant find any info , and i dont family with js , some try run test dont work .... 
so is there any extension or just config to use  this lsp in vim ?

**Comment by WillDawnlll** [comment]
https://code.visualstudio.com/blogs/2022/08/16/markdown-language-server

**Comment by fannheyward** [comment]
https://github.com/microsoft/vscode/tree/main/extensions/markdown-language-features/server Markdown Language Server can't use in other editor by now. Maybe an extension is needed to make it work.
